pycbc/distributions/power_law.py

```diff
@@ -134,7 +134,7 @@ def __init__(self, dim=None, **params):
             self._norm *= self.dim  / \
                                    (self._bounds[p][1]**(self.dim) -
                                     self._bounds[p][0]**(self.dim))
-            self._lognorm = numpy.log(self._norm)
+            self._lognorm += numpy.log(self._norm)
```

Comment:
Oh, I missed this. No, undo this. You'll be overcounting the norm for multiple parameters here.

Actually, as I look at it, if you want to improve the code, take the `self._lognorm` out of the loop. There's no reason to recompute the lognorm for every parameter. So, the block of code should be:
```
        self._norm = 1.0
        for p in self._params:
            self._norm *= self.dim  / \
                                   (self._bounds[p][1]**(self.dim) -
                                    self._bounds[p][0]**(self.dim))
        self._lognorm = numpy.log(self._norm)
```

Reply:
This is fixed and improved according to your suggestion in the [latest commit ](https://github.com/gwastro/pycbc/pull/3681/commits/275a1818197b48367ece03c881b089668dbc1f2d).